docs/deployment/vm.mdx

```diff
@@ -91,5 +90,6 @@ To do that, you'll need to set these environment variables when running your Bri
 - `POSTGRES_DATABASE`: The name of the database to use within Postgres.
 - `POSTGRES_USERNAME`: The username Briefer should use to connect to your Postgres instance.
 - `POSTGRES_PASSWORD`: The password for the chosen Postgres user.
+- `POSTGRES_PRISMA_URL`: The connection URL for your Postgres instance in the format `postgresql://<username>:<password>@<host>:<port>/<database>`.
```

Comment:
I think this reveals that we have another problem. We should actually construct this env var ourselves using the other ones.

This is what we're doing in the `docker-compose` file:
https://github.com/briefercloud/briefer/blob/58a857effd83dc78cd92304a9c1aa3faa4569a9e/docker-compose.yaml#L29

We need to fix this:
https://github.com/briefercloud/briefer/blob/58a857effd83dc78cd92304a9c1aa3faa4569a9e/docker/setup/setup.py#L115
Make this use the `POSTGRES_` variables when they are provided instead of the localhost db.

Reply:
Hi @vieiralucas, thanks for reviewing!

 This absolutely makes sense! It should be possible to easily construct the `POSTGRES_PRISMA_URL` from the other environmental variables. 
 
 Below is the syntax example that I used for my instance:
postgresql://postgres.<user>:[YOUR-PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:5432/**postgres**

As you can see the mistake I made is to not name my database `briefer` but leaving it with the standard setting `postgres`. It seems to work fine for now. 

I think to avoid this mistake in the future, the two options I see are:

1. Specifying in the /docs that the database name has to be briefer
2. Replacing `briefer?schema=public` with the environment variable name from the user input

Let me know how you'd like to proceed

Reply:
Oh! That explains, that's definitely a bug on our end as-well. We should use the `POSTGRES_DATABASE` provided by the user, it should not be mandatory to use `briefer`.
I'll fix this in this same branch before merging the PR. Thank you again @sachstimo, you're very helpful!

Reply:
Of course! Happy to help :) Keep up the great work with Briefer 🫶